src/jit/lowerxarch.cpp

```diff
@@ -2300,58 +2300,170 @@ void Lowering::ContainCheckSIMD(GenTreeSIMD* simdNode)
 //  Arguments:
 //     containingNode - The hardware intrinsic node which contains 'node'
 //     node - The node to check
+//     supportsRegOptional - On return, true if 'containingNode' supports regOptional operands; otherwise, false.
```

Comment:
I would suggest that you explicitly say this is an "out" argument.

Reply:
Fixed.